WMFModel/WMFArticle+CoreDataProperties.h

```diff
@@ -0,0 +1,29 @@
+#import "WMFArticle+CoreDataClass.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFArticle (CoreDataProperties)
+
++ (NSFetchRequest<WMFArticle *> *)fetchRequest;
+
+@property (nonatomic) BOOL isBlocked;
+@property (nullable, nonatomic, copy) NSString *key;
+@property (nullable, nonatomic, copy) NSDate *viewedDate;
+@property (nullable, nonatomic, copy) NSDate *viewedCalendarDate;
+@property (nullable, nonatomic, copy) NSString *viewedFragment;
+@property (nonatomic) double viewedScrollPosition;
+@property (nullable, nonatomic, copy) NSDate *newsNotificationDate;
+@property (nullable, nonatomic, copy) NSDate *savedDate;
+@property (nonatomic) BOOL wasSignificantlyViewed;
+@property (nullable, nonatomic, copy) NSString *displayTitle;
+@property (nullable, nonatomic, copy) NSString *wikidataDescription;
+@property (nullable, nonatomic, copy) NSString *snippet;
+@property (nullable, nonatomic, copy) NSString *thumbnailURLString;
+@property (nonatomic) double latitude;
```

Comment:
Is the extra info on CLLocation used anywhere? Will we ever need to fetch by lat/lon?